### DISCUSSION TITLE
Manually declaring dependent cells

### DISCUSSION THREAD

**Discussion by haakon-e** [fonsp/Pluto.jl#2933] [Questions about using Pluto] [answered]
Is there a way to "tell" Pluto that a cell should depend on another cell? In the attached, recorded example, I set the global theme for Makie in one cell, then create a plot in another. If I do this naively, i.e. 
```julia
# cell 1:
Makie.set_theme!( ... )

# cell 2:
begin
heatmap(...)
current_figure()
end
```
then, updating the arguments to `set_theme!` does nothing to the plot. However, if I assign a dummy-variable `a` (note, `set_theme!` returns `nothing`), that I also put into the figure block, things update nicely (see screen recording). Is there a (more) proper way to do this?

https://github.com/fonsp/Pluto.jl/assets/45243236/b0a7e6d0-8ce1-465f-8c68-a016260c3706

**Comment by pankgeorg** [comment, marked as answer]
This is exactly the way to do it. I would add a more declarative name to the theme; e.g.: `rerun_on_theme_change = Makie.theme` and then 
```julia
begin
  rerun_on_theme_change;
  heatmap(...)
end
```

But that's really just a variable :D

**Reply by haakon-e** [reply]
Thanks!
